Find a functionally equivalent code:android.view.ViewGroup.offsetChildrenTopAndBottom:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Offset the vertical location of all children of this view by the specified number of pixels.
 *
 * @param offset the number of pixels to offset
 *
 * @hide
 */

@@ -6,17 +6,18 @@
         final View v = children[i];
         v.mTop += offset;
         v.mBottom += offset;
-        if (v.mDisplayList != null) {
+        if (v.mRenderNode != null) {
             invalidate = true;
-            v.mDisplayList.offsetTopAndBottom(offset);
+            v.mRenderNode.offsetTopAndBottom(offset);
         }
     }
     if (invalidate) {
         invalidateViewProperty(false, false);
     }
+    notifySubtreeAccessibilityStateChangedIfNeeded();
 }

